deepchem/models/tensorgraph/tests/test_layers.py

```diff
@@ -875,3 +875,16 @@ def test_IRV(self):
       assert out_tensor.shape == (batch_size, n_tasks)
       irv_reg = IRVRegularize(irv_layer, 1.)()
       assert irv_reg.eval() >= 0
+
+  def test_hingeloss(self):
```

Comment:
Could you also add a test that the new Hinge Loss layer can be pickled? (Should be added in https://github.com/deepchem/deepchem/blob/master/deepchem/models/tensorgraph/tests/test_layers_pickle.py with other tests for pickling)